src/app/recipes/recipes.component.ts

```diff
@@ -37,7 +45,11 @@ export class RecipesComponent implements AfterViewInit {
     switch(recipeSelection.event) {
       case RecipeItemEventType.Delete:
         return this._recipeDialogService.openDeleteRecipeDialog().pipe(
-          switchMap(isConfirmed => isConfirmed ? this._recipesService.delete(id) : EMPTY)
+          switchMap(isConfirmed => isConfirmed
+            ? this._recipesService.delete(id).pipe(
```

Comment:
Same as above. You should create action DeleteRecipe (which will fetch data with serivce) and DeleteRecipeSuccess (which will set data in store).